tests/app/main/views/test_users.py

```diff
@@ -289,27 +260,6 @@ def _return_user_export_response(self, data_api_client, framework, users, framew
         )
 
     ##########################################################################
```

Comment:
We could delete this ugly line of `################` while we're here maybe?

Reply:
👍